services/virtualmachine/service.go

```diff
@@ -57,7 +57,13 @@ func (s *service) ProcessQuery(ctx context.Context, input *services.ProcessQuery
 	logger := s.logger.WithTags(trace.LogFieldFrom(ctx))
 
 	if input.BlockHeight != 0 {
-		panic("Run local method with specific block height is not yet supported")
+		err := errors.New("Run local method with specific block height is not yet supported")
+		return &services.ProcessQueryOutput{
+			CallResult:              protocol.EXECUTION_RESULT_ERROR_INPUT,
```

Comment:
my thought is to use the protocol.ArgumentsArrayEmpty().Raw() instead of just empty byte array and also to return the last committed block height and time even if there is nothing that can be done with it. 
please also change the other error in this func to have the argumens empty thing. thanks

Reply:
Done